.circleci/test-scripts/baseline/format.awk

```diff
@@ -3,6 +3,6 @@ BEGIN {
     print "|---|---|---|---|---|---|---|"
 }
 {
-    warn = ($7<=0.05) ? " :question:" : "";
+    warn = ($7==1) ? " :question:" : "";
     printf "|%s|%s|%s|%s|%s|%s|%s%s|%s",$1,$2,$3,$4,$5,$6,$7,warn,ORS
```

Comment:
[ultra-nit] Since this is a true/false check now, maybe we can do something like this?
```suggestion
    warn = ($7==1) ? ":green_circle:" : ":red_circle:";
    printf "|%s|%s|%s|%s|%s|%s|%s|%s",$1,$2,$3,$4,$5,$6,warn,ORS
```